.github/workflows/deploy-staging.yaml

```diff
@@ -235,7 +235,6 @@ jobs:
       - id: deploy-template-sns
         name: Deploy CloudFormation stack for SNS topic
         uses: aws-actions/aws-cloudformation-github-deploy@v1
-        if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
I discovered this when testing the first time. An issue arise when I triggered this staging from another branch, not master. This causes staging to be applied, but throw an error in the UI pod because it can not find the Cognito client. I dont' see a reason why this should only be limited to master. This workflow is only used creating a staging environment, not modifying it, so we want the environment to work and relevant resources to be created.